src/routes/events/containers/EventsContainer.js

```diff
@@ -10,29 +10,28 @@ import createNavigateToEvent from '../../../modules/app/createNavigateToEvent'
 import type { Dispatch } from 'redux'
 import type { StoreActionType } from '../../../modules/app/StoreActionType'
 import createNavigateToIntegreatUrl from '../../../modules/app/createNavigateToIntegreatUrl'
-import { branch, renderComponent } from 'recompose'
-import EventNotAvailableContainer from './EventLanguageNotAvailableContainer'
-import { Failure } from '../../../modules/error/components/Failure'
+import type { NavigationScreenProp } from 'react-navigation'
+import withError from '../../../modules/error/hocs/withError'
 import withTheme from '../../../modules/theme/hocs/withTheme'
 
-const mapStateToProps = (state: StateType, ownProps) => {
-  const {resourceCache, eventsRouteMapping, languages, language, city} = state.cityContent
+type OwnPropsType = {|
+  navigation: NavigationScreenProp<*>
+|}
 
-  if (languages && !languages.map(languageModel => languageModel.code).includes(language)) {
-    return {invalidLanguage: true}
-  }
+const mapStateToProps = (state: StateType, ownProps: OwnPropsType) => {
+  const {resourceCache, eventsRouteMapping, city} = state.cityContent
 
-  if (eventsRouteMapping.errorMessage !== undefined) {
+  if (eventsRouteMapping.errorMessage !== undefined || resourceCache.errorMessage !== undefined) {
     return {error: true}
```

Comment:
So, do we ignore the `errorMessage` for now? We might want to display it, don't we?

Reply:
There is a separate task to display error codes (https://issues.integreat-app.de/browse/NATIVE-193). I think displaying "cryptic" error message (which these error messages are) does not increase the usability.